packages/react-native-cli/src/commands/OnPremiseUrlsCommand.ts

```diff
@@ -0,0 +1,48 @@
+import prompts, { PromptObject } from 'prompts'
+import { UrlType, OnPremiseUrls } from '../lib/OnPremise'
+import onCancel from '../lib/OnCancel'
+
+export default async function run (...requiredUrls: UrlType[]): Promise<OnPremiseUrls> {
```

Comment:
out of interest why did you go for a spread here rather than take a single array argument?

Reply:
No real reason; I think it looks neater when it's the only parameter & saves 2 characters but I don't think there's an actual advantage